include/pgduckdb/logger.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+namespace pgduckdb {
+
+/* PG Error level codes */
+#define DEBUG5		10
+#define DEBUG4		11
+#define DEBUG3		12
+#define DEBUG2		13
+#define DEBUG1		14
+#define LOG			15
+#define INFO		17
+#define NOTICE		18
+#define WARNING		19
+#define PGWARNING	19
+#define WARNING_CLIENT_ONLY	20
+#define ERROR		21
+#define PGERROR		21
+#define FATAL		22
+#define PANIC		23
```

Comment:
Shouldn't we comment these out/remove them. We don't want people to use ERROR and higher with this function right?

Reply:
Maybe even assert somewhere that `WARNING_CLIENT_ONLY` is the highest used code.